project_task_pull_request_state/models/project_task.py

```diff
@@ -0,0 +1,109 @@
+# Copyright Cetmix OU 2023
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0).
+from odoo import api, fields, models
+
+
+class ProjectTaskState(models.Model):
+    _inherit = "project.task"
+
+    pr_state = fields.Selection(
+        selection=lambda self: self._selection_pr_state(),
+        track_visibility="onchange",
+        copy=False,
+        string="PR State",
+        compute="_compute_pr_state",
+        readonly=False,
+    )
+
+    def _selection_pr_state(self):
+        """Function to select the state of the pull request"""
+        return [
+            ("open", "Open"),
+            ("draft", "Draft"),
+            ("merged", "Merged"),
+            ("closed", "Closed"),
+        ]
+
+    @api.model
+    def create(self, vals):
```

Comment:
Use multi record "create" implementation with vals_list instead of vale and api.create multi decorator

Reply:
resolved